ros2bag/ros2bag/api/__init__.py

```diff
@@ -125,6 +125,17 @@ def check_positive_float(value: Any) -> float:
         raise ArgumentTypeError('{} is not the valid type (float)'.format(value))
 
 
+def check_not_negative_float(value: Any) -> float:
```

Comment:
the reason for introducing this is, check_positive_float need to stay with `ros2 bag play --rate`, right? in that case 0 should not be taken as an argument.

Reply:
That is indeed the [second usage of the `check_positive_float`](https://github.com/ros2/rosbag2/blob/e01e1efe57c4f226eed21e2eefb0b459fd3bd0b8/ros2bag/ros2bag/verb/play.py#L51), in this case, zero does not make sense.